code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -32,3 +31,73 @@
 	if (src.use_sound)
 		playsound(src.loc, src.use_sound, 50, 1, -5)
 	..()
+
+/obj/item/weapon/storage/bible/proc/Set_Religion(mob/user)
+	if(use_check(user))
+		return
+
+	if(!ishuman(user))
+		return
+
+	var/religion_name = "Christianity"
+	var/new_religion = sanitize(input(user, "You are the crew services officer. Would you like to change your religion? Default is Christianity, in SPACE.", "Name change", religion_name), MAX_NAME_LEN)
+
+	if (!new_religion)
+		new_religion = religion_name
+
+	var/book_name = "holy bible"
+	book_name = sanitize(input(user, "Would you like the change your bible name? Default is holy bible.", "Book name change", book_name), MAX_NAME_LEN)
+
+	if (!book_name)
+		book_name = book_name

```

Comment:
Thiiis doesn't work. If line 49 results in null, the end result would still be null. Best way to do this would be as follows:
```dm
	var/book_name = sanitize(input(user, "Would you like the change your bible name? Default is holy bible.", "Book name change", book_name), MAX_NAME_LEN)

	if (!book_name)
		book_name = "holy bible"
```